scripts/setup-node.sh

```diff
@@ -0,0 +1,279 @@
+██  █  ██ ███████ ██████  ██ ██  ██ ██ ██ ██  ██ ██   ███ ██
+██ ███ ██ ██   ██ ██   ██ ██  ██ ██ ██ ██  ██ ██ ██    ██
+ ███ ███  ██   ██ ██   ██ ██   ████ ██ ██   ████  ██████  ██
+  ${NOFORMAT}
+  "
+}
+
+print_axelar() {
+    msg "
+  ${GREEN}
+ █████  ██   ██ ███████ ██       █████  ██████
+██   ██  ██ ██  ██      ██      ██   ██ ██   ██
+███████   ███   █████   ██      ███████ ██████
+██   ██  ██ ██  ██      ██      ██   ██ ██   ██
+██   ██ ██   ██ ███████ ███████ ██   ██ ██   ██
+  ${NOFORMAT}
+  "
+}
+
+reset_chain() {
+    print_warning
+    msg "${RED}RESET CHAIN FLAG USED!!!${NOFORMAT}"
+    local directories
+    local reply
+    if [[ ! -d $root_directory ]]; then
+        msg "root directory $root_directory doesn't exist"
+        return
+    fi
+    directories="$(ls -ah "$root_directory")"
+    msg "list of files/folders in root directory:"
+    msg "$directories"
+    msg "${RED}WARNING! You are about to reset the entire chain state${NOFORMAT}"
+    msg "${RED}your current state will be backed up to $root_directory.bak${NOFORMAT}"
+    msg "${RED}you can manually delete the backed state by running 'rm -rf $root_directory.bak'${NOFORMAT}"
+    msg "${RED}to proceed type 'understood'${NOFORMAT}"
+    read -r reply
+    if [[ "$reply" = "understood" ]]; then
+        msg "moving previous state from $root_directory to $root_directory.bak"
+        rm -rf "$root_directory.bak"
+        mv "$root_directory" "$root_directory.bak"
+    else
+        msg "invalid input. Exiting..."
+        exit 1
+    fi
+}
+
+create_directories() {
+    msg "creating required directories"
+    if [[ ! -d "$root_directory" ]]; then mkdir -p "$root_directory"; fi
+    if [[ ! -d "$config_directory" ]]; then mkdir -p "$config_directory"; fi
+}
+
+copy_configuration_files() {
+    if [ ! -f "${config_directory}/genesis.json" ]; then
+        msg "Copying genesis file to the config directory"
+        cp "${resources}/genesis.json" "${config_directory}/genesis.json"
+    else
+        msg "genesis file already exists"
+    fi
+
+    if [ ! -f "${config_directory}/seeds.toml" ]; then
+        msg "Copying seeds.toml to the config directory"
+        cp "${resources}/seeds.toml" "${config_directory}/seeds.toml"
+    else
+        msg "seeds.toml file already exists"
+    fi
+
+    if [ -f "${config_directory}/config.toml" ]; then
+        msg "backing up existing config.toml and overwriting it"
+        cp "${config_directory}/config.toml" "${config_directory}/config.toml.backup"
+    fi
+
+    echo "copying config.toml"
+    cp "${git_root}/configuration/config.toml" "${config_directory}/config.toml"
+
+    if [ -f "${config_directory}/app.toml" ]; then
+        msg "backing up existing app.toml and overwriting it"
+        cp "${config_directory}/app.toml" "${config_directory}/app.toml.backup"
+    fi
+
+    msg "copying app.toml"
+    cp "${git_root}/configuration/app.toml" "${config_directory}/app.toml"
```

Comment:
Why overwrite `config.toml` and `app.toml`?  This has been a source of confusion in the past.  Perhaps we should treat it more like `genesis.json` and `seeds.toml` --- only copy it if it doesn't already exist.

Reply:
Some of our config values are still fluctuating (such as min gas price), so we want to override them by default for now, and then we'll just warn it or let users decide.